Formula/netpbm.rb

```diff
@@ -46,6 +47,7 @@ def install
       s.change_make_var! "ZLIB", "-lz"
       s.change_make_var! "JASPERLIB", "-ljasper"
       s.change_make_var! "JASPERHDR_DIR", "#{Formula["jasper"].opt_include}/jasper"
+      s.gsub! "/usr/local/netpbm/rgb.txt", "#{opt_prefix}/misc/rgb.txt"
```

Comment:
Why `opt_prefix` and not `prefix`?

Reply:
Force of habit, sorry.

Reply:
Thanks for updating the path.  I like that better than adding an extra link as I had originally suggested.